apps/studio/package.json

```diff
@@ -19,7 +19,7 @@
     "postinstall": "electron-builder install-app-deps",
     "postuninstall": "electron-builder install-app-deps",
     "release:prs": "./build/get-prs.sh",
-    "test:integration": "TEST_MODE=1 ELECTRON_RUN_AS_NODE=1 yarn electron ../../node_modules/jest/bin/jest.js --config ./jest.integration.config.js"
```

Comment:
Not sure why this was deleted? This is needed to have the tests run in the right way

Reply:
Oh, I guess this doesn't work on Windows right?

Reply:
I'll fix that, I removed that to try to get tests working locally on Windows